node_follower/src/txn_service.rs

```diff
@@ -1,23 +1,31 @@
-use crate::{Result, Settings};
+use crate::{Error, Result, Settings};
 use helium_proto::{
     services::{
         transaction::{self, TxnQueryReqV1, TxnQueryRespV1, TxnSubmitReqV1, TxnSubmitRespV1},
         Channel,
     },
     BlockchainTxn,
 };
+use rand::{seq::SliceRandom, thread_rng};
 
-type TransactionClient = transaction::Client<Channel>;
+pub type TransactionClient = transaction::Client<Channel>;
+pub type TransactionClients = Vec<TransactionClient>;
 
 #[derive(Debug, Clone)]
 pub struct TransactionService {
     client: TransactionClient,
+    clients: Option<TransactionClients>,
```

Comment:
I primarily kept it this way to avoid changing any other service other than injector. There isn't much overhead afaict because if the `clients` is not configured (nothing other than injector would configure it) its just None.

Reply:
a comment on each would provide context, useful for a year later when we all forget why both fields are there

Reply:
Added a comment [0150308](https://github.com/helium/oracles/pull/278/commits/0150308c651a1f29ddb94ed54a37787e6fd77e34)